docs/readmes/lte/extended_5G_sa_features.md

```diff
@@ -0,0 +1,173 @@
+
+![Stateless Feature](assets/lte/Stateless_feature.png?raw=true "Stateless Feature")
+
+## GTP extension header support
+
+The Extension Header Length field specifies the length of the particular Extension header in 4 octet units. The Next Extension Header Type field specifies the type of any Extension Header that may follow a particular Extension Header. If no such Header follows, then the value of the
+Next Extension Header Type shall be 0.
+
+```text
+	 0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+     | Ext-Hdr Length|                                               |
+     +-+-+-+-+-+-+-+-+                                               |
+     |                  Extension Header Content                     |
+     .                                                               .
+     .                                               +-+-+-+-+-+-+-+-+
+     |                                               |  Next-Ext-Hdr |
+     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+```
+
+In 5G SA, each flow is forwarded based on the appropriate QoS rules. QoS rules are configured by SMF as QoS profiles to UP components and these components perform QoS controls to PDUs based on rules. In downlink, a pipelineD pushes QFI into an extension header, and transmits the PDU to RAN.  In uplink, each UE obtains the QoS rule from SMF, and transmits PDUs with QFI containing the QoS rules to the RAN.
+
+## Additional AGW Configurations
+
+### Network initiated Session Modification
+
+Stub CLI used to add the QoS policy in Network initiated session modification.
+
+```text
+session_manager_cli.py policy_rar IMSI001019976545660 IMSI001019976545660-112383 INTERNET ADD 10 6 "UL,6,192.168.128.12,0,192.168.200.1,12345;DL,1,8.8.8.8,0,192.168.128.12,0" "1000000,2000000,1000000,2000000,15,1,0"
+```
+
+Stub CLI used to delete the QoS policy in Network initiated session modification.
+
+```text
+session_manager_cli.py policy_rar IMSI001019976545660 IMSI001019976545660-112383 INTERNET DEL 10 6 "UL,6,192.168.128.12,0,192.168.200.1,12345;DL,1,8.8.8.8,0,192.168.128.12,0" "1000000,2000000,1000000,2000000,15,1,0"
+```
+
+Here is the details of the [Stub CLI command](assets/lte/Session_modification_stub_cli_commands "Stub CLI Command") for add or delete the QoS policy in Network initiated session modification.
+
+### Stateless feature
+
+To support stateless feature first should enable config_stateless flag.
+
+```text
+magmad_cli.py config_stateless enable	
+```
+
+To check the status of config_stateless flag.
+
+```text
+magmad_cli.py config_stateless check_stateless
+```
+
+The above command should give `AGW Mode: STATELESS`.
+
+## Common Issues and Troubleshooting
+
+GTP tunnel not added in OVS table
+
+- Description :-

```

Comment:
`:-` looks funny. What is that supposed to do?
Rather use `Description: Your description text here`.

Reply:
Addressed Review Comment